src/core/formatters.ts

```diff
@@ -216,6 +216,7 @@ namespace Plottable.Formatters {
           return d3.time.format(timeFormat[i].format)(d);
         }
       }
+      return undefined;
```

Comment:
can you change timeFormat[7] to the default case and return this at the end instead of `undefined` functionally the same but makes the default case easier to find